exo-go/src/process_helpers/index.go

```diff
@@ -0,0 +1,45 @@
+package processHelpers
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os/exec"
+
+	shellwords "github.com/mattn/go-shellwords"
+)
+
+// Run runs the given command, waits for the process to finish and
+// returns the output string and error (if any)
+func Run(command, dir string) (string, error) {
+	commandWords, err := shellwords.Parse(command)
+	if err != nil {
+		return "", err
+	}
+	cmd := exec.Command(commandWords[0], commandWords[1:]...) // nolint gas
+	cmd.Dir = dir
+	outputArray, err := cmd.CombinedOutput()
+	output := string(outputArray)
+	return output, err
+}
+
+// Start runs the given command in the given dir directory, and returns
+// the pointer to the command, stdout pipe, output buffer and error (if any)
+func Start(command string, dir string) (*exec.Cmd, io.WriteCloser, *bytes.Buffer, error) {
+	commandWords, err := shellwords.Parse(command)
+	if err != nil {
+		return nil, nil, &bytes.Buffer{}, err
+	}
+	cmd := exec.Command(commandWords[0], commandWords[1:]...) // nolint gas
```

Comment:
This is the only `gas` rule that I know where we disable it. (We do it twice in this file and thats it). And it is for a good reason as thats specifically how we are testing our CLI.